src/Cake/Scripting/Roslyn/Stable/RoslynScriptSessionFactory.cs

```diff
@@ -10,15 +10,15 @@ namespace Cake.Scripting.Roslyn.Stable
 {
     using Core.IO;
```

Comment:
Looks like this always been here, @patriksvensson any special  reason for separating the usings?